circuits/src/zk_circuits/proof_linking.rs

```diff
@@ -0,0 +1,218 @@
+//! Helpers for linking proofs between circuits
+//!
+//! TODO: Figure out which proving and verifying keys to use once we generate
+//! them for the circuits
```

Comment:
FWIW, I think it's fair and safe to assume / assert that all circuits use the same SRS, of sufficient degree for each. We could cache this SRS (similar to `TESTING_SRS`) and extract the KZG commit / open keys directly from it.

Would make the review of these helpers simpler, wouldn't have to think about "is this the right commit / open key"

In fact, this assumption is already implicit, because we use wiring polynomial commitments from different circuits to construct the linking identity polynomial commitment

Reply:
it is a matter of length, the proving key for one circuit is a trimmed version of the common SRS. I'll leave the todo for now, but agree it is likely trivially obviated.